### ISSUE TITLE
Update Main.sublime-menu

### ISSUE THREAD

**Issue by thecodechef** [jgbishop/sxs-settings#7]
``` json
"args": { "file": "${packages}/sxs_settings/sxs_settings.sublime-settings" }
```

to:

``` json
"args": { "file": "${packages}/SxS Settings/sxs_settings.sublime-settings" }
```

**Comment by jgbishop** [comment]
Thanks for catching this bug! I'm waiting on a rename pull request (the name of this plugin will be changing slightly), and once that's done I'll merge this.

**Comment by jgbishop** [comment]
I just released 1.2.1 that fixes this issue.
